client/my-sites/pages/helpers.js

```diff
@@ -48,5 +48,11 @@ export const sortPagesHierarchically = ( pages ) => {
 		insertChildren( topLevelPage.ID, 1 );
 	} );
 
+	// Places the Homepage at the top of the list.
+	if ( homepageId !== 0 ) {
+		const homepage = sortedPages.findIndex( ( page ) => page.ID == homepageId );
```

Comment:
Small linting issue here:

<img width="401" alt="Screen Shot 2021-08-16 at 15 19 36" src="https://user-images.githubusercontent.com/1233880/129570168-0960af07-15a2-4894-a68e-b906a8b76efa.png">

```suggestion
		const homepage = sortedPages.findIndex( ( page ) => page.ID === homepageId );
```